CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## [1.3.0](https://github.com/Workiva/over_react_codemod/compare/1.2.0...1.3.0)
+
+- Add a flag --no-partial-upgrades to Component2 codemod executable that will prevent partial component upgrades from occurring
+
+- Fix a bug that could occur when parsing a pubsec version in `dependency_overrides` section.
+
+
+## [1.2.0](https://github.com/Workiva/over_react_codemod/compare/1.1.0...1.2.0)
+
+- Add `react16_upgrade` codemod
+
+  - Fix compatibility issues common in react15 code
+  - Update version upper bound of react and over_react in pubspec.yaml to
+    allow for incoming react16 updates
+
+- Add `component2_upgrade` codemod
+
+  - Migrates components to `UiComponent2` (coming in OverReact 3.1.0)
```

Comment:
I know we usually spell the Library OverReact, but when referring to the package (like on line 13), don't we usually do lowercase with the dash thing?
```suggestion
  - Migrates components to `UiComponent2` (coming in over_react 3.1.0)
```